File: GerenciaMusic360/ClientApp/app/shared/models/project-task.ts
import { UserTask } from './user-task';
export interface ProjectTask {
    completed: boolean;
    estimatedDateVerfication: string;
    estimatedDateVerficationString: any;
    id: number;
    notes: string;
    position: number;
    projectId: number;
    required: boolean;
    templateTaskDocumentDetailId: number;
    templateTaskDocumentDetailName: string;
    templateTaskDocumentId: number;
    templateTaskDocumentName: string;
    users: UserTask[];
    selectedUsers: UserTask[];
    isNew: boolean;
    checked: boolean;
    comments: boolean;
    hasContractPending: number;
}

export interface ProjectTaskDetail extends ProjectTask {
    year: any;
    day: any;
    weekDay: any;
    month: any;
}

File: GerenciaMusic360.Entities/ProjectTask.cs
using System;
using System.Collections.Generic;

namespace GerenciaMusic360.Entities
{
    public partial class ProjectTask
    {
        public int Id { get; set; }
        public int ProjectId { get; set; }
        public string Notes { get; set; }
        public bool Completed { get; set; }
        public int TemplateTaskDocumentDetailId { get; set; }
        public DateTime EstimatedDateVerfication { get; set; }
        public DateTime Created { get; set; }
        public string Creator { get; set; }
        public DateTime? Modified { get; set; }
        public string Modifier { get; set; }
        public bool Required { get; set; }
        public bool Status { get; set; }
        public string EstimatedDateVerficationString { get; set; }
        public int TemplateTaskDocumentId { get; set; }
        public string TemplateTaskDocumentName { get; set; }
        public string TemplateTaskDocumentDetailName { get; set; }
        public short Position { get; set; }
        public int? ProjectIdRelation { get; set; }
        public long UserProfileId { get; set; }
        public string UserProfileName { get; set; }
        public bool UserProfileAuthorized { get; set; }
        public int ConfigurationTaskId { get; set; }
        public int? DepartmentId { get; set; }
        public string DepartmentName { get; set; }
        public List<ConfigurationTaskAutorize> Users { get; set; }
        public int hasContractPending { get; set; }
        public string CalendarNotes { get; set; }
        public string  ProjectName { get; set; }
    }
}
